### ISSUE TITLE
MQTT-Presence: Accept boolean as onMessage / offMessage

### ISSUE THREAD

**Issue by georg90** [wutu/pimatic-mqtt#45]
Hi,

my Xiaomi presence sensors report their state via a boolean (true/false). Could we enable to accept boolean as onMessage/offMessage?

```
MQTT publish: topic 'zigbee2mqtt/0x00158d0001ad4694', payload '{"occupancy":true,"linkquality":99}'
```

Else we could cast the incoming message to string?

Thanks a lot!

**Comment by mwittig** [comment]
Hi, I think the problem is that the MqttPresence sensor can only deal with simple payloads (e.g. "true", "false" strings) right now, and it is not possible to use a certain property inside a JSON payload.

**Comment by mwittig** [comment]
I have tested your scenario and as far as I can see it works as expected with the new JSON parse feature

**Comment by mwittig** [comment]
released pimatic-mqtt@0.9.13

Please re-open if you're still seeing the issue
